pkgs/servers/nosql/ferretdb/default.nix

```diff
@@ -5,7 +5,7 @@
 
 buildGoModule rec {
   pname = "ferretdb";
-  version = "0.7.1";
+  version = "0.8.0";
```

Comment:
Changing the version means that the SRI hashes for the GitHub fetch (line 14) and the Go dependencies (line 22) need to change accordingly as well.